requirements.txt

```diff
@@ -4,7 +4,7 @@ biolinkml==1.4.9
 certifi==2020.12.5
 chardet==3.0.4
 click==7.1.2
-elasticsearch==7.5.1
+elasticsearch==7.12.0
```

Comment:
This change was necessary to resolve the exception `ImportError: cannot import name 'decodestring' from 'base64'` while using python 3.9.2 on my local host